### PR TITLE
:seedling: Included directories that don't require coverage

### PR DESCRIPTION
**Pull request by naveensrinivasan**
- Included directories that don't require coverage.

#### What kind of change does this PR introduce?

(Is it a bug fix, feature, docs update, something else?)

- [ ] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the new behavior (if this is a feature change)?**

- [ ] Tests for the changes have been added (for bug fixes/features)

#### Does this PR introduce a user-facing change?

For user-facing changes, please add a concise, human-readable release note to
the `release-note`

(In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note

```

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -7,7 +7,16 @@ codecov:
...
   - "cron/**/*"
   - "clients/mockclients/**/*"
-  - "attestor/command/**/*"
+  # ignoring them as these are part of the CLI tool.
+  - "attestor/command/check.go"
+  - "attestor/command/cli.go"
+  - "attestor/command/sign.go"
+  - "attestor/root.go"
+  # ignoring them as these are internal tools for generating docs.
+  - "docs"
```
Comment (spencerschrock) [inline .codecov.yml:16]: It seems the pattern to ignore a folder should be:
"docs/**/*"

https://docs.codecov.com/docs/ignoring-paths

File: .codecov.yml [hunk 2]
```diff
@@ -7,7 +7,16 @@ codecov:
 ignore:
   - "cron/**/*"
   - "clients/mockclients/**/*"
-  - "attestor/command/**/*"
+  # ignoring them as these are part of the CLI tool.
```
Comment (spencerschrock) [inline .codecov.yml:10]: Should we be ignoring the CLI aspects?

Comment (naveensrinivasan) [reply .codecov.yml]: Took care of it.

File: .codecov.yml (hunk 1 above)
Comment (naveensrinivasan) [reply .codecov.yml]: 👍 Thanks

**Review by spencerschrock — APPROVED** [review]

**Pull request merged**